radicale/tests/test_base.py

```diff
@@ -766,6 +766,14 @@ def test_authorization(self):
         assert status == 207
         assert "href>/user/<" in answer
 
+    def test_authentication(self):
+        """Test if server sends authentication request."""
+        self.configuration.set("rights", "type", "owner_only")
+        self.application = Application(self.configuration, self.logger)
```

Comment:
Changing `self.application` in the test is a bit odd, maybe we should put this test elsewhere.